roles/vendor/redis/tasks/set_redis_facts.yml

```diff
@@ -0,0 +1,7 @@
+- name: Ansible collector
+  setup:
+    filter: 'ansible_memtotal_mb'
+
+- name: Set mem for redis
+  set_fact:
+    redis_mem: "{{ ((ansible_memtotal_mb*(redis_maxmem/100))*1024*1024)|int }}"
```

Comment:
NICE! We need to be careful though! 

There's 2 Redis nodes running on each machine, and during production testing I've found repeated OOM-crashes, at times causing both master and slave of the same shards to reside on one server!